package.json

```diff
@@ -96,7 +96,8 @@
     "tsx": "^4.6.1",
     "typedoc": "^0.25.3",
     "typescript": "~5.2.0",
-    "verdaccio": "^5.24.1"
+    "verdaccio": "^5.24.1",
+    "@graphql-codegen/plugin-helpers": "^3.1.1"
```

Comment:
nit: please put this in alphabetical order (or closest possible if we don't have strict ordering yet in this file).